entity/dag/entity_actions.go

```diff
@@ -258,3 +257,23 @@ func Remove(def Definition, repo repository.ClockedRepo, id entity.Id) error {
 
 	return nil
 }
+
+// RemoveAll delete all Entity items for all identities
+func RemoveAll(def Definition, repo repository.ClockedRepo) error {
+	localIds, err := ListLocalIds(def, repo)
+	if err != nil {
+		return err
+	}
+	for _, id := range localIds {
+		fmt.Println("removing bug " + id.String())
```

Comment:
You can't have anything printing from the core. If some information is worthy enough, that should be bubbled up in the form of an event, and each UI can elect to display it in the form that make sense.
Here, I think we can just remove it.

Reply:
Removed.